views/problem_display.ejs

```diff
@@ -53,7 +54,22 @@
     <footer>
         <% include partials/footer %>
     </footer>
-    
+    <script>
+    function copyToClipboard(id) {
+    const string = id.innerHTML;
```

Comment:
Check below, this should be str instead of string.

You should run and test and then make a commit.